src/universal/components/RetroReflectPhase/ReflectionTypeColumn.js

```diff
@@ -0,0 +1,177 @@
+
+import AddReflectionButton from 'universal/components/AddReflectionButton/AddReflectionButton';
+import ReflectionCard from 'universal/components/ReflectionCard/ReflectionCard';
+import AnonymousReflectionCard from 'universal/components/AnonymousReflectionCard/AnonymousReflectionCard';
+import withAtmosphere from 'universal/decorators/withAtmosphere/withAtmosphere';
+import CreateReflectionMutation from 'universal/mutations/CreateReflectionMutation';
+import RemoveReflectionMutation from 'universal/mutations/RemoveReflectionMutation';
+import UpdateReflectionIsEditingMutation from 'universal/mutations/UpdateReflectionIsEditingMutation';
+import UpdateReflectionContentMutation from 'universal/mutations/UpdateReflectionContentMutation';
+import deserialize from 'universal/utils/draftjs/deserialize';
+import serialize from 'universal/utils/draftjs/serialize';
+import getNextSortOrder from 'universal/utils/getNextSortOrder';
+import ui from 'universal/styles/ui';
+
+// Helpers
+
+const forPhaseItem = (retroPhaseItemId: string, reflections: Array<TeamFragment>) => (
+  reflections
+    .filter((reflection) => reflection.retroPhaseItemId === retroPhaseItemId)
+);
+
+// Actions
+
+const handleClickAddReflection = (
+  environment: Environment,
+  meetingId: string,
+  retroPhaseItemId: string,
+  reflections: Array<RetroReflection>
+) => {
+  CreateReflectionMutation(environment, {
+    meetingId,
+    retroPhaseItemId,
+    sortOrder: getNextSortOrder(reflections)
+  });
+};
+
+const handleDelete = (environment: Environment, reflectionId: string, meetingId: string) => {
+  RemoveReflectionMutation(
+    environment,
+    {reflectionId},
+    meetingId
+  );
+};
+
+const handleSave = (environment: Environment, reflectionId: string, meetingId: string, editorState: EditorState) => {
+  const content = serialize(editorState.getCurrentContent());
+  UpdateReflectionContentMutation(
+    environment,
+    {reflectionId, content},
+    meetingId
+  );
+};
+
+const handleStartEditing = (environment: Environment, reflectionId: string, meetingId: string) => {
+  UpdateReflectionIsEditingMutation(
+    environment,
+    {reflectionId, isEditing: true},
+    meetingId
+  );
+};
+
+const handleStopEditing = (environment: Environment, reflectionId: string, meetingId: string) => {
+  UpdateReflectionIsEditingMutation(
+    environment,
+    {reflectionId, isEditing: false},
+    meetingId
+  );
+};
+
+// Components
+
+const ColumnWrapper = styled('div')({
+  display: 'flex',
+  flexDirection: 'column'
+});
+
+const ReflectionsArea = styled('div')({
+  flexDirection: 'column',
+  overflow: 'auto'
+});
+
+const TypeDescription = styled('div')({
+  fontSize: '1.2rem',
+  fontWeight: 'bold'
+});
+
+const TypeHeader = styled('div')({
+  marginBottom: '2rem'
+});
+
+const TypeTitle = styled('div')({
+  color: ui.labelHeadingColor
+});
+
+type Props = {
+  atmosphere: Environment,
+  retroPhaseItem: RetroPhaseItem,
+  team: Team
```

Comment:
this should be TeamFragment (ideally we just rename TeamFragment to Team to reduce confusion).
By using the flow type that relay gives us, it'll throw really useful errors. for example, if i tried to call `team.name`, that exists on Team, but does _not_ exist on TeamFragment. It's super.